tests/qos/qos_helpers.py

```diff
@@ -1,7 +1,221 @@
+from ansible_host import AnsibleHost
+from netaddr import IPAddress, IPNetwork
+from qos_fixtures import lossless_prio_dscp_map, conn_graph_facts, leaf_fanouts
+import json
+import re
+import ipaddress
+
+PFC_GEN_FILE = 'pfc_gen.py'
+PFC_GEN_LOCAL_PATH = '../../ansible/roles/test/files/helpers/pfc_gen.py'
+PFC_GEN_REMOTE_PATH = '~/pfc_gen.py'
+
+def atoi(text):
+    return int(text) if text.isdigit() else text
+
+def natural_keys(text):
+    return [atoi(c) for c in re.split(r'(\d+)', text)]
+
+def ansible_stdout_to_str(ansible_stdout):
+    """
+    @Summary: The stdout of Ansible host is essentially a list of unicode characters. This function converts it to a string.
+    @param ansible_stdout: stdout of Ansible 
+    @return: Return a string  
+    """
+    result = ""
+    for x in ansible_stdout:
+        result += x.encode('UTF8')
+    return result
+
 def eos_to_linux_intf(eos_intf_name):
     """
     @Summary: Map EOS's interface name to Linux's interface name
     @param eos_intf_name: Interface name in EOS
     @return: Return the interface name in Linux
     """
-    return eos_intf_name.replace('Ethernet', 'et').replace('/', '_') 
+    return eos_intf_name.replace('Ethernet', 'et').replace('/', '_')
+
+def get_phy_intfs(host_ans):
```

Comment:
This function should be moved to tests/common/platform/interface_utils.py

But do it with another PR.